djstripe/models.py

```diff
@@ -497,10 +497,11 @@ def sync_current_subscription(self, cu=None):
             return sub_obj
 
     def update_plan_quantity(self, quantity, charge_immediately=False):
+        stripe_subscription = self.stripe_customer.subscription
+        if not stripe_subscription:
+            raise SubscriptionUpdateFailure("Customer does not have a subscription with Stripe")
```

Comment:
I did respond, but the comment is hidden because the diff is outdated.

The problem is that, as currently written, sync_current_subscription won't actually do anything in this case, although maybe it should.